apps/opik-backend/src/main/java/com/comet/opik/api/BatchDelete.java

```diff
@@ -13,5 +13,5 @@
 @Builder(toBuilder = true)
 @JsonIgnoreProperties(ignoreUnknown = true)
 @JsonNaming(PropertyNamingStrategies.SnakeCaseStrategy.class)
-public record TracesDelete(@NotNull @Size(min = 1, max = 1000) Set<UUID> ids) {
+public record BatchDelete(@NotNull @Size(min = 1, max = 1000) Set<UUID> ids) {
```

Comment:
Can we confirm with SDK if this rename will not break anything from their side?

Reply:
This is a POJO for handling the requests payload. It shouldn't have any effect over the API.

Reply:
The payload is the same but if `TracesDelete` object is referred by the SDK, it will break. I recommend running the `generate_openapi.sh`. If the SDK or end to end tests are covering this path, it should break. You can test it by quickly running the Github actions e.g: by pushing a draft PR.